proto/store/project.proto

```diff
@@ -42,3 +44,25 @@ message ProtectionRule {
   }
   BranchSource branch_source = 5;
 }
+
+message Binding {
+  // The role that is assigned to the members.
+  // Format: roles/{role}
+  string role = 1;
+
+  // Specifies the principals requesting access for a Bytebase resource.
+  // For users, the member should be: user:{email}
```

Comment:
// For users, the member should be: users/{userUID}
  // For groups, the member should be: groups/{email}

Reply:
Good catch